io.github.simple64.simple64.yaml

```diff
@@ -0,0 +1,72 @@
+app-id: io.github.simple64.simple64
+runtime: org.kde.Platform
+runtime-version: "6.3"
+sdk: org.kde.Sdk
+command: gui-wrapper.sh
+rename-icon: simple64
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --talk-name=org.freedesktop.ScreenSaver
+modules:
+  - shared-modules/libusb/libusb.json
+
+  - name: 7zip
+    buildsystem: simple
+    build-commands:
+      - install -D 7zz -t /app/bin
+      - ln -s /app/bin/7zz /app/bin/7za
+    sources:
+      - type: archive
+        url: https://www.7-zip.org/a/7z2107-linux-x64.tar.xz
```

Comment:
you are still building this flatpak on `aarch64` tough. IMHO you should build it from source instead.